### PR TITLE
Guide for linking installations

### PR DESCRIPTION
**Pull request by fredrikbaberg**
I've started on a guide for linking two installations. I noticed several questions about this on Discord, and I had some of my own, so thought it would be good to have some more documentation on this.

Note that:
- I haven't tested to build the documentation (I couldn't get yarn to work right now).
- There are no images, just a basic description.
- It's currently in markdown as I haven't used mdx before, let me know if I should use mdx instead.

### CODE REVIEW

**Comment by fredrikbaberg** [comment]
I've now added images and made sure this builds, so I've removed WIP.

I hope this can help someone. Let me know if something needs to be changed.

File: docs/guides/unmanic_link_installations.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+### Requirements
+
+* At least two installs of Unmanic:
+    * One install will be called `main`, this has access to a folder with files to process.
+    * The other install will be called `remote`, this has to be reachable from `main` but does not require access to the files. Files will automatically be transfered from `main` to `remote` and back.
+
+### Setup
+
+* On `main`, setup a [library](/docs/configuration/libraries/adding_libraries.mdx) following the normal instructions. Let us call the library `Guide`.
```
Comment (Josh5) [inline docs/guides/unmanic_link_installations.md:24]: Change this to `/docs/configuration/libraries/adding_libraries`

File: docs/guides/unmanic_link_installations.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+* On `remote`, create a new library. The name of this library has to be **exactly** the same as the library in `main`, in this case `Guide`. You can either export the library from `main` and import it on `remote`, see tip below, or set it up manually. This library only needs plugins for making changes, i.e. encoding. "Scanning" plugins, such as filter by size, are not needed.
+
+:::note
+Library name must be identical on both installations.
+:::
+:::note
+It is recommended to create a new library for this, instead of using the default library.
+:::
+
+* [Link](/docs/configuration/linking.mdx) the installs:
```
Comment (Josh5) [inline docs/guides/unmanic_link_installations.md:35]: Change this to `/docs/configuration/link_settings`

**Comment by Josh5** [comment]
Looks good. Thanks!

The way that these markdown docs work with [Docusaurus](https://docusaurus.io/) is that the link points to the page ID (like how you have authored this document's id as `unmanic_link_installations` in 
```
---
title: Linking Unmanic Installations
description: Linking Unmanic Installations
id: unmanic_link_installations
---
```

File: docs/guides/unmanic_link_installations.md (hunk 1 above)
Comment (fredrikbaberg) [inline docs/guides/unmanic_link_installations.md:24]: Should this be changed to `[library](adding_libraries)`, and similar for other references?

Comment (Josh5) [reply docs/guides/unmanic_link_installations.md:24]: It should be 

`[Library](/docs/configuration/libraries/adding_libraries)`

File: docs/guides/unmanic_link_installations.md (hunk 2 above)
Comment (Josh5) [reply docs/guides/unmanic_link_installations.md:35]: This should be 

`[Link](/docs/configuration/link_settings)`

**Review by Josh5 — APPROVED** [review]
I'll make these mentioned changes. Sorry, I never completed this review

**Pull request merged**